docroot/profiles/humsci/su_humsci_profile/src/HumsciContentAccessUpdater.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Drupal\su_humsci_profile;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\node\NodeInterface;
+
+/**
+ * Service for updating content access settings.
+ *
+ * The content access module allows different permissions per content type. The
+ * configuration (content_access.settings) is config ignored. This service will
+ * update relevent access setting per bundle or per node without affecting any
+ * existing settings.
+ */
+class HumsciContentAccessUpdater {
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
```

Comment:
It does not look like this is ever used and can be removed from the service entirely.

Reply:
👍 Service removed from class and services.yml file.